### DISCUSSION TITLE
need help for configuring nvim-cmp

### DISCUSSION THREAD

**Discussion by RiteshChepuri** [hrsh7th/nvim-cmp#1736] [Q&A]
I'm trying to make transition from packer to lazy every thing worked fine in packer without a problem but when i'm trying to configure in lazy the issue rises as failed to run config for nvim-cmp
`return {
  "hrsh7th/nvim-cmp",
  event = "InsertEnter",
  dependencies = {
    "hrsh7th/cmp-buffer", -- Source for text in buffer
    "hrsh7th/cmp-path", -- Source for file system path
    "L3MON4D3/LuaSnip", -- Snippet Engine
    "saadparwaiz1/cmp_luasnip", -- Snippet Engine
    "rafamadriz/friendly-snippets", -- Useful Snippets
    "onsails/lspkind.nvim", -- VS Code Like pictograms
  },
  config = function()
    local cmp = require("cmp")
    local luasnip = require("luasnip")
    local lspkinf = require("lspkind")

    -- Loads VS Code style snippets from installed Plugins (e.g. friendly snippets)
    require("luasnip.loaders.from_vscode").lazy_load()

    cmp.setup({
      completion = {
        completeopt = "menu,menuone,preview,noselect",
      },
      snippet = { -- configure how nvim-cmp interacts with snippet engine
      expand = function(args)
        luasnip.lsp_expand(args.body)
      end,
      },
      mapping = cmp.mapping.preset.insert({
        ["<C-k>"] = cmp.mapping.select_prev_item(), -- previous suggestion
        ["<C-j>"] = cmp.mapping.select_next_item(), -- next suggestion
        ["<Tab>"] = cmp.mapping.select_next_item(), -- next suggestion
        ["<C-b>"] = cmp.mapping.scroll_docs(-4),
        ["<C-f>"] = cmp.mapping.scroll_docs(4),
        ["<C-Space>"] = cmp.mapping.complete(), -- show completion suggestions
        ["<C-e>"] = cmp.mapping.abort(), -- close completion window
        ["<CR>"] = cmp.mapping.confirm({ select = false }), -- Confirm selection
        ["<ESC>"] = cmp.mapping.abort(), -- Close completion window
      }),

      -- Sources for Autocompletion
      sources = cmp.config.sources({
        -- { name = "nvim_lsp"},
        { name = "luasnip"}, -- Snippets
        { name = "buffer"}, -- Buffer
        { name = "path"}, -- File system Paths
      }),

      -- Configure lspkind for vs-code like pictograms in completion menu
      formatting = {
        format = lspkind.cmp_format({
          maxwidth = 50,
          ellipsis_char = "...",
        }),
      },
    })
  end,
}
`
![image](https://github.com/hrsh7th/nvim-cmp/assets/81766942/9b631318-8654-4a72-995f-74cae2595adc)

How do I fix this.

**Comment by fitrh** [comment]
I think you just misspelled `lspkind` to `lspkinf`

```diff
 config = function()
   local cmp = require("cmp")
   local luasnip = require("luasnip")
-  local lspkinf = require("lspkind")
+  local lspkind = require("lspkind")
```

**Reply by RiteshChepuri** [reply]
Thank you, My Bad.
